### PR TITLE
Refactor some models to yaml, refactor of create-model, incl documentation

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
move some object to separate objects and reduce height of in_front_of areas

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
changes to robotics_testlabs tested on AMIGO

File: models/robotics_testlabs/battery_table/model.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+color: {red: 0.4, green: 0.3, blue: 0.2}
+#areas:
```
Comment (jlunenburg) [inline models/robotics_testlabs/battery_table/model.yaml]: Remove commented stuff

File: models/robotics_testlabs/flight_case/model.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+color: {red: 0.1, green: 0.1, blue: 0.1}
+#areas:
+#-   name: near
+    #offset: 0.7
```
Comment (jlunenburg) [inline models/robotics_testlabs/flight_case/model.yaml]: Remove commented stuff

File: scripts/create-model.py [hunk 3]
```diff
@@ -1,277 +1,363 @@
...
 
-    #def set_type(self, t):
-    #    """ Sets the type of the model
-    #    :param t: string with the type of the model
-    #    """
-    #    self.data['type'] = t
+    # def set_type(self, t):
+    #     """
+    #     Sets the type of the model
+    #     :param t: the type of the model
```
Comment (jlunenburg) [inline scripts/create-model.py]: Remove? Or comment why this should remain in place?

**Pull request merged**